packages/react-performance/README.md

```diff
@@ -17,7 +17,36 @@ $ yarn add @shopify/react-performance
 
 The most basic way to use the tools in this package is to record information and display it locally to the user. In practice you usually only want to do this in development so that developers can easily see performance information.
 
-#### Display data using NavigationListener
+### Adding some `<PerformanceMark />`s
+
+We will need to add some `<PerformanceMark />` components to our React application in order to help `@shopify/react-performance` determine when your application is fully loaded and React has mounted your components.
+
+```tsx
+//Cats.tsx
+import React from 'react';
+import {PerformanceMark} from '@shopify/react-performance';
+
+/**
+ * Represents an entire page of our application
+ */
+export function Cats() {
+  return (
+    <section>
+      <h1>Cats</h1>
+      <ul>
+        <li>Kokusho</li>
+        <li>Moe</li>
+        <li>Smoky</li>
+      </ul>
+      <PerformanceMark stage="complete" id="cats" />
+    </section>
+  );
+}
+```
+
+In production apps, it is important to always have one `<PerformanceMark stage="complete" />` on each page level component. Otherwise, client-side navigations may not be recorded properly.
+
```

Comment:
Maybe we need to add an example for `<PerformanceMark stage="usable"/>`

Reply:
We have that below, titled `### Tracking _usable_ vs _complete_`

Reply:
Yes, please ignore this comment, I realized that later on.